tamr_unify_client/project/attribute_mapping/resource.py

```diff
@@ -5,8 +5,9 @@ class AttributeMapping:
     (ex: /projects/1/attributeMappings/1), but these types of URLs do not exist for attribute mappings
     """
 
-    def __init__(self, data):
+    def __init__(self, data, client=None):
```

Comment:
For consistency with the other resources, the order here should be `client, data, alias=None`. This would go into the "breaking changes" section of the changelog